Userland/Utilities/nologin.cpp

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2022, Beckett Normington <beckett@b0ba.dev>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/File.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+
+ErrorOr<int> serenity_main(Main::Arguments) {
+    TRY(Core::System::pledge("stdio rpath"));
+
+    auto file_or_error = Core::File::open("/etc/nologin", Core::OpenMode::ReadOnly);
+    if (file_or_error.is_error()) {
+        outln("This account is currently not available.");
```

Comment:
I don't have the serenity source pulled up currently, but if those functions are able to use StringViews (`"blah"sv`) instead of normal literal strings, it should probably use those instead.

Reply:
Okay, I believe I have changed this.